.devcontainer/devcontainer.json

```diff
@@ -1,9 +1,9 @@
 {
-    "image":  "ghcr.io/microsoft/msquic/linux-build-xcomp",
+    "dockerFile": "Dockerfile",
     "hostRequirements": {
         "cpus": 4,
         "memory": "2gb"
     },
-    "onCreateCommand": "pwsh -c './scripts/prepare-machine.ps1 -ForOneBranch -InitSubmodules'",
-    "runArgs": ["--privileged"]
+    "onCreateCommand": "pwsh -c './scripts/prepare-machine.ps1'",
```

Comment:
Why remove "-ForOneBranch"?

Reply:
Where is the "-ForOneBranch" used?

Reply:
The reason why our devcontainer uses a docker image in the first place is we can just load up the docker image and skip the step of `./scripts/prepare-machine.ps1` and build the code right away.

Reply:
I don't see any if-else branch in script though

Reply:
if (!$ForOneBranch -and !$ForOneBranchPackage -and !$ForBuild -and !$ForTest -and !$InstallXdpDriver -and !$UninstallXdp) {

Reply:
https://github.com/microsoft/msquic/blob/aaecb0fac5a3902dd244df8a05942ffc35a6729f/scripts/prepare-machine.ps1#L112

Reply:
oooooh it is ALL false case.....
But setting -ForOneBranch doesn't set -ForTest and -ForBuild.
If my understanding is correct, user will run prepare-machine.ps1 sooner or later, because this codespace settings is for manual operation. not test automation, right?

Reply:
For building the project, the user won't need to run prepare-machine.ps1 because everything is set up in the docker image already and its ready to build.